webdav/src/main/java/ch/cyberduck/core/dav/DAVSession.java

```diff
@@ -150,19 +150,12 @@ public void login(final Proxy proxy, final LoginCallback prompt, final CancelCal
                     new UsernamePasswordCredentials(host.getCredentials().getUsername(), host.getCredentials().getPassword()));
             client.setCredentials(provider);
             if(preferences.getBoolean("webdav.basic.preemptive")) {
-                switch(proxy.getType()) {
-                    case DIRECT:
-                    case SOCKS:
-                        // Enable preemptive authentication. See HttpState#setAuthenticationPreemptive
-                        client.enablePreemptiveAuthentication(host.getHostname(),
-                                host.getPort(),
-                                host.getPort(),
-                                Charset.forName(preferences.getProperty("http.credentials.charset"))
-                        );
-                        break;
-                    default:
-                        client.disablePreemptiveAuthentication();
-                }
+                // Enable preemptive authentication. See HttpState#setAuthenticationPreemptive
+                client.enablePreemptiveAuthentication(host.getHostname(),
```

Comment:
No need anymore for the proxy the type check? Guess it is (or was) necessary to support authenticating proxies. Was added in 096877af2f4a2dcbae18a2cac134a40ac4843033.

Reply:
See [28fafaf](https://github.com/iterate-ch/cyberduck/pull/15876/commits/28fafaf12bb0df3e2db662742121cd3a081ecfeb). It reverts commit https://github.com/iterate-ch/cyberduck/commit/ff9c41cc85894d328d8725cc0af790a2604111bd. Proxy authentication uses `Proxy-Authentication` request header. I do not understand why this was made dependent on the proxy configuration.

Reply:
Successfully tested against an authenticating proxy.